LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/client/LabClient.java

```diff
@@ -314,4 +316,50 @@ private String getLabSecretName(final String credentialVaultKeyName) {
         final String[] parts = credentialVaultKeyName.split("/");
         return parts[parts.length - 1];
     }
+
+    /**
+     * Enable CA/Special Policies for any Locked User.
+     * Enable Policy can be used for GlobalMFA, MAMCA, MDMCA, MFAONSPO, MFAONEXO.   Also test users can have more than 1 policy assigned to the same user.
+     *
+     * @param upn    Enter a valid Locked User UPN (optional)
+     * @param policy Enable Policy can be used for GlobalMFA, MAMCA, MDMCA, MFAONSPO, MFAONEXO. (optional)
+     * @return boolean value indicating policy enabled or not.
+     */
+    public boolean enablePolicy(@NonNull final String upn, @NonNull final String policy) {
```

Comment:
Let's change the policy parameter from String to the enum. I believe the ProtectionPolicy enum or if that doesn't fit then create a new enum for this purpose

Reply:
addressed it in 1967e8c7bc9664f31c21697a0145fb14ff79e370